dumpling/tests/composite_string_key_large/run.sh

```
@@ -0,0 +1,147 @@
+  exit 1
+fi
+if [ "$translations_chunks" -lt 2 ]; then
+  echo "FAIL: translations produced $translations_chunks chunks, expected >= 2"
+  exit 1
+fi
+
+# Each data chunk must be a self-contained INSERT (per lance6716 review).
+# The filename pattern $DB_NAME.<table>.<seq>.sql excludes -schema.sql
+# meta files (which legitimately have no INSERT statement).
+for chunk in $(find "$DUMPLING_OUTPUT_DIR" -maxdepth 1 -type f \
+    \( -name "$DB_NAME.events.[0-9]*.sql" -o -name "$DB_NAME.translations.[0-9]*.sql" \)); do
```

Shellcheck (0.11.0)

[warning] For loops over find output are fragile. Use find -exec or a while read loop.

(SC2044)